src/electionguard-ui/ElectionGuard.UI/ViewModels/ViewKeyCeremonyViewModel.cs

```diff
@@ -12,14 +13,17 @@ public partial class ViewKeyCeremonyViewModel : BaseViewModel
     private KeyCeremonyMediator? _mediator;
 
     private bool _joinPressed;
+    private ILogger<ViewKeyCeremonyViewModel> _logger;
```

Comment:
noted before. Should be ILogger, not ILogger<T>. Alternatively... should we consider having `_logger` be in the base class and we pass in the specific ILogger<T> needed?

Reply:
moved to base class